bin/memory-profiling/prometheus.yml

```diff
@@ -0,0 +1,7 @@
+global:
+  scrape_interval: 15s
+
+scrape_configs:
+  - job_name: 'cadvisor'
+    static_configs:
+      - targets: ['172.17.0.3:8080']
```

Comment:
Note to self: Make this configurable. `172.17.0.3` should be the IP address docker assigns to the `cAdvisor` process.